documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -5,127 +5,257 @@
+If multiple different Kafka Connect instances are used, these settings must be unique for the workers of each Kafka Connect cluster created.
 
-[discrete]
-== Example YAML showing Kafka Connect configuration
+.Worker configuration that must be unique for each Kafka Connect instance
 [source,yaml,subs="attributes+"]
 ----
 apiVersion: {KafkaConnectApiVersion}
 kind: KafkaConnect
-metadata:
-  name: my-connect
+# ...
 spec:
-  # ...
   config:
     group.id: my-connect-cluster
     offset.storage.topic: my-connect-cluster-offsets
     config.storage.topic: my-connect-cluster-configs
     status.storage.topic: my-connect-cluster-status
-  # ...
 ----
 
-[discrete]
-== Connectors
+The group ID identifies the Kafka Connect worker nodes. Kafka Connect clusters cannot share the ID or topics as it will create errors.
+The names of the connectors used by each Kafka Connect instance must also be unique.
 
-Connectors are configured separately from Kafka Connect.
-The configuration describes the source input data and target output data to feed into and out of Kafka Connect.
-The external source data must reference specific topics that will store the messages.
+You can also link:{BookURLDeploying}#creating-new-image-using-kafka-connect-build-str[deploy connector plugins^] using the `build` configuration of the `KafkaConnect` resource.
+Connector plugins provide specific configuration for the creation of connector instances.
+
+After connector plugins have been added to the Kafka Connect deployment, you can use Strimzi’s `KafkaConnector` custom resource or the Kafka Connect API to manage connector instances.
+You can also create new connector instances using these options.
 
-Kafka provides two built-in connectors:
+The Cluster Operator manages Kafka Connect clusters deployed using the `KafkaConnect` resource and connectors created using the `KafkaConnector` resource.
 
---
-* `FileStreamSourceConnector` streams data from an external system to Kafka, reading lines from an input source and sending each line to a Kafka topic.
-* `FileStreamSinkConnector` streams data from Kafka to an external system, reading messages from a Kafka topic and creating a line for each in an output file.
---
+[discrete]
+== Connector plugin configuration
+
+Connector plugins provide the configuration for connecting to an external data source.
+Plugins contain a set of files that define the implementation required to connect to the data source.
+Connector plugins for many external systems are available for use with Kafka Connect.
+You can also create your own connector plugins.
 
-You can add other connectors using connector plugins, which are a set of JAR files or TGZ archives that define the implementation required to connect to certain types of external system.
+Connector plugins provide the configuration as a set of files that define the implementation required to connect to specific types of external data system.
```

Comment:
I don't actually think this sentence (line 100) is necessary. The earlier sentences explain the concepts well.